Envfile

```diff
@@ -73,6 +67,7 @@ variable :ALGOLIASEARCH_SEARCH_ONLY_KEY, :String, default: only_in_test
 variable :AWS_ID, :String, default: "Optional"
 variable :AWS_SECRET, :String, default: "Optional"
 variable :AWS_BUCKET_NAME, :String, default: "Optional"
+variable :AWS_UPLOAD_REGION, :String, default: "Optional"
```

Comment:
this has to be an empty string, for the `OR` in the carrierwave initializer and the site map to work. Otherwise the app locally will boot with "Optional" as the region

`Optional` is just an arbitrary string I hope we can get rid of :D

Reply:
Oh snap! Good catch!